guarddog/analyzer/sourcecode/serialize-environment.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+  - id: serialize-environment
+    message: |
```

Comment:
Was this used in a specific malware or are we just considering it suspicious?

Reply:
I found it in a malware analysis https://www.mend.io/wp-content/media/2022/02/Mend-npm-Threat-Repot.pdf